test/optimize/test_optimize.py

```diff
@@ -158,6 +159,8 @@ def get_optimizer(library, solver):
     elif library == 'scipydiffevolopt':
         optimizer = optimize.ScipyDifferentialEvolutionOptimizer(
             options=options)
+    elif library == 'pyswarms':
```

Comment:
the options update in the pyswarms optimizer has to check whether options exist (see there, as otherwise just setting some options as here will fail)

Reply:
I am not sure I know what you mean with this comment...
what exactly should I change here?

Reply:
here nothing, just in the class as mentioned above, cause otherwise this here doesn't work